Visual Studio Code Dev Container fails build

The dev container in visual studio code fails to build the project with the latest from master.

## Expected Behavior
Succeeds build.

## Actual Behavior
The following error appears:
```bash
root@ea9dfb9a7c2f:/workspaces/keda# make build
GOROOT is not defined
Makefile:61: recipe for target 'checkenv' failed
make: *** [checkenv] Error 1
```

## Steps to Reproduce the Problem

  1. Pull the latest from master
  2. Using Visual Studio Code, open project as a remote container
  3. Run `make build`

## Specifications

  - **KEDA Version:** master
  - **Platform & Version:** Running this on Windows 10

--- a/Makefile
+++ b/Makefile
@@ -58,8 +58,7 @@ GO_BUILD_VARS= GO111MODULE=on CGO_ENABLED=$(CGO) GOOS=$(TARGET_OS) GOARCH=$(ARCH
 .PHONY: checkenv
 checkenv:
 ifndef GOROOT
-	@echo "GOROOT is not defined"
-	@exit 1
+	@echo "WARNING: GOROOT is not defined"
 endif
 
 .PHONY: build

